azure-iot-device/azure/iot/device/iothub/inbox_manager.py

```diff
@@ -29,6 +29,7 @@ def __init__(self, inbox_type):
         self.input_message_inboxes = {}
         self.generic_method_request_inbox = self._create_inbox()
         self.named_method_request_inboxes = {}
+        self.twin_inbox = self._create_inbox()
```

Comment:
>twin_inbox [](start = 13, length = 10)

`twin_patch_inbox` for clarity I think.

Also, do we need to be concerned about Twin Patches being received out of order? I know IoTHub has to have a mechanism to handle it's received patches being out of order.

Reply:
If we start applying the patches and creating a single Twin object that represents the current state of everything, then we will have to worry about out-of-order patches.  That can be built on top of this.  It doesn't belong here because we can't define a policy for handling out-of-order patches without that higher view.  

---
In reply to: [296340599](https://github.com/Azure/azure-iot-sdk-python-preview/pull/114#discussion_r296340599) [](ancestors = 296340599)